### DISCUSSION TITLE
emails in the automation part

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2364] [Help]
Hi there,
I want to add the automation part to my project. I have tried to send a SMTP email when a row is deleted in a particular table, but I haven't made it out. Maybe I am doing something wrong here. 
Can someone help me out with that, at least to show me each step that I have to do in order to make it happen?
Thanks,
Era.

**Comment by kevmodrome** [comment]
Hi Era,

Have you set up SMTP correctly?

**Comment by shogunpurple** [comment]
Hi @erakastrati - do you have your SMTP server set up and ready to go? Who is your provider? Here are the steps:

### Configure your SMTP Settings
- Go to account portal
- Visit the "Email" tab
- Fill in your SMTP credentials from your server. Budibase should actually verify this, so it will fail if your server is not set up correctly.
![Screenshot 2021-08-16 at 10 29 47](https://user-images.githubusercontent.com/11256663/129542306-aac0ac93-4c7b-4edb-a8fa-d301c0d8d01e.png)

## Set up your automation
- Set up delete row trigger for the relevant table
- Set up SMTP email action
![Screenshot 2021-08-16 at 10 30 55](https://user-images.githubusercontent.com/11256663/129542456-98b2239d-5a41-49d7-8ece-0d645ab9478d.png)

If you have the above set up and it is still not working, we can debug from there.

Thanks!

**Comment by erakastrati** [comment]
<img width="1610" alt="Screen Shot 2021-08-16 at 12 43 01 PM" src="https://user-images.githubusercontent.com/74623805/129551755-f4af4b53-c78f-4fd8-af4d-d2039abe07aa.png">
Hey @shogunpurple and @kevmodrome, thanks for the response.
I just set up the STP informations. Is this the next step. If so,  I can't select the 'use an existing automation' part.

**Reply by erakastrati** [reply]
<img width="1161" alt="Screen Shot 2021-08-16 at 12 59 33 PM" src="https://user-images.githubusercontent.com/74623805/129553692-1ecf85c2-83ff-42d6-ab94-249395865f88.png">
Now it is working to use an existing automation part, but the email is not coming to the inbox. 
@shogunpurple @kevmodrome do you have time for a call? I have got some questions.
